app/render.py

```diff
@@ -17,6 +17,35 @@
 ]
 
 
+def get_url_for_alert(alert, alerts):
+    """
+    Gets the url slug for an alert (given the global alerts object obtained from `Alerts.load()`.)
+
+    Note: The alert must be public to have a url slug.
+
+    Will return a date string in the style '3-jun-2021', unless there were already alerts that day, in which case it'll
+    append a 1-indexed counter on to the end eg '3-jun-2021-2'. Note that the first alert of the day never has a count.
+    """
+    alerts_on_this_day = sorted([
+        other_alert
+        for other_alert in alerts.public
```

Comment:
Question (possibly for @quis / @karlchillmaid )

We're only assigning URLs for public alerts (severe/government channel).

If we ever want specific URLs for other alerts (operator/test/rmt/exercise etc), then we need to make that decision _before_ we send both a public and a test alert on the same day. (As the URLs will be affected because they're numbered). After that point we'll need to stick with this URL scheme. We could always have operator alerts be on a different naming scheme entirely, eg `/alerts/operator/25-may-2021`.

The decision doesn't need to happen right now, but thought it was worth flagging in case we ever want to have individual pages for operator tests.

Reply:
Unlikely that’d we’d give non-public alerts a URL, but a prefix/suffix/different structure feels right.

Reply:
Agree

Reply:
RMT "test" alerts are public and should be published as they'll be received by real users.